From 05887e5e74395db2ef59d59b41668820e6bb9e74 Mon Sep 17 00:00:00 2001
From: Ivan Lopes <lopesivan.ufrj@gmail.com>
Date: Mon, 4 Aug 2025 20:18:25 -0300
Subject: [PATCH] custom

---
 main.c | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 main.c

diff --git a/main.c b/main.c
new file mode 100644
index 0000000..4266390
--- /dev/null
+++ b/main.c
@@ -0,0 +1,10 @@
+/* hello.c: A standard "Hello, world!" program */
+
+#include <stdio.h>
+
+int main(int argc, char *argv[])
+{
+    printf("Hello, world!\n");
+
+    return 0;
+}
-- 
2.50.1

