From 54b54066345064c5f367c64f88bb308cbe99b53c Mon Sep 17 00:00:00 2001
From: Ivan Lopes <lopesivan.ufrj@gmail.com>
Date: Sat, 27 Apr 2024 02:53:13 -0300
Subject: [PATCH] custom

---
 INSTALL | 1 +
 1 file changed, 1 insertion(+)

diff --git a/INSTALL b/INSTALL
index 84536ca..d6d3f8b 100644
--- a/INSTALL
+++ b/INSTALL
@@ -25,3 +25,4 @@ It's suggested that you try
 
 as well.
 
+brew install tdl@42
-- 
2.41.0

