From 0c19587138fc36eda1d0164a9d95a3d82d113dec Mon Sep 17 00:00:00 2001
From: Ivan Lopes <lopesivan.ufrj@gmail.com>
Date: Wed, 9 Aug 2023 01:24:53 -0300
Subject: [PATCH] custom

---
 configure.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.sh b/configure.sh
index 8e359cf..012c848 100755
--- a/configure.sh
+++ b/configure.sh
@@ -29,6 +29,8 @@
 # ----------------------------------------------------------------------------
 # Run!
 
+OPT=${HOME}/developer/bash-config
+
 files=(
   "bashrc"
   "bash_logout"
@@ -40,7 +42,7 @@ for f in ${files[*]}; do
   test -L ~/.${f} &&
     rm ${HOME}/.${f}
 
-  ln -s ${HOME}/developer/bash-config/${f} ${HOME}/.${f}
+  ln -s ${OPT}/${f} ${HOME}/.${f}
 done
 
 # ----------------------------------------------------------------------------
-- 
2.41.0

